### DISCUSSION TITLE
Can tauri ios apps be put on  Apple app store?

### DISCUSSION THREAD

**Discussion by zwl1619** [tauri-apps/tauri#6060] [Q&A]
Can tauri ios apps be put on  Apple app store?

**Comment by osbre** [comment]
https://github.com/tauri-apps/tauri-docs/issues/691

**Comment by FabianLars** [comment]
We don't tested it ourselves yet tbh. For android we recommended flutter's guide so i guess it should hopefully also work for ios: <https://docs.flutter.dev/deployment/ios>
